hack/dci-merge.sh

```diff
@@ -71,17 +71,23 @@ for PR in $PRS; do
         # extract TestBos2 commands
         if grep -qE "^\s*TestBos2:\s*" <<< "$DESC"; then
             # shellcheck disable=SC2001,SC2086
-            CMD+=("$(sed -ne 's/^\s*TestBos2:\s*//p' <<< $DESC)")
```

Comment:
why not `sed -ne 's/^\s*TestBos2:\s*//p; s/\r$//'` ?

Reply:
@fredericlepied this is the only way I've found to do this in one single line: `sed -ne 's/^\s*TestBos2:\s*//p' <<< $DESC | sed 's/\r$//'`
Using other options like yours or `sed -ne 's/^\s*TestBos2:\s*//p' <<< $DESC | sed 's/\r$//'` are not working

Reply:
Interesting, at first glance both options seem to return the same but I see the option from Fred is rendering oddly:

```
[dciteam@jumphost ~]$ DESC=$(curl -s https://api.github.com/repos/redhatci/ansible-collection-redhatci-ocp/pulls/333 | jq -r .body)                          
[dciteam@jumphost ~]$ echo "$DESC"
TestBos2: virt

Some of them was bug some of them change to be more ansible format and some of them changed to modules
                                                   
[dciteam@jumphost ~]$ ONE=$(sed -ne 's/^\s*TestBos2:\s*//p' <<< $DESC | sed 's/\r$//')
[dciteam@jumphost ~]$ TWO=$(sed -ne 's/^\s*TestBos2:\s*//p' -e 's/\r$//' <<< $DESC)
[dciteam@jumphost ~]$ echo ${ONE}
virt
[dciteam@jumphost ~]$ echo ${TWO}
virt
[dciteam@jumphost ~]$ echo ${ONE}manny
virtmanny
[dciteam@jumphost ~]$ echo ${TWO}manny
manny
```

Reply:
Yes, and apart from this, it's still embedding the `\r` character, which is the reason why it's not working afterwards when trying to schedule a pipeline with that code.
I've refactored my code to do it in one line. Since there are no more reviews with alternatives to improve this, and don't know if @fredericlepied will be able to check this during this week since he's off, I'll propose to merge this and then improve the code if needed in a separate change.